src/Components/RankingTable.js

```diff
@@ -0,0 +1,66 @@
+import React, { useMemo } from 'react'
```

Comment:
For the next sprint, please remove the unused columns, and make it work with the API call.